charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -324,17 +269,7 @@ spec:
               value: $(HOST_IP):8502
             {{- end }}
           command:
-            - /consul-bin/consul
-            - connect
-            - envoy
-            - -gateway=terminating
-            - -proxy-id=$(POD_NAME)
-            {{- if $root.Values.global.enableConsulNamespaces }}
-            - -namespace={{ default $defaults.consulNamespace .consulNamespace }}
-            {{- end }}
-            {{- if $root.Values.global.adminPartitions.enabled }}
-            - -partition={{ $root.Values.global.adminPartitions.name }}
-            {{- end }}
+            - sleep infinity
```

Comment:
This will be replaced with the startup for consul dataplane in a future PR into this branch.